lib/Watching.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file
+ * @copyright  2016 Matt Smith (Development Seed)
+ * @license    Licensed under {@link https://spdx.org/licenses/BSD-3-Clause-Clear.html BSD-3-Clause-Clear}.
+ *             Github.js is freely distributable.
+ */
+
+import Requestable from './Requestable';
+import debug from 'debug';
+const log = debug('github:watching');
+
+/**
+ * Watching a Repository registers the user to receive notifications on new discussions, as well as events in the user's
+ * activity feed.
+ */
+export default class Watching extends Requestable {
```

Comment:
At the moment we use `module.exports` for exporting classes. I'm not sure about the reason as this was made by @clayreimann but I think it's because we aren't using the babel plugin to revert the export of defaults to the behavior of Babel 5.